### ISSUE TITLE
State is not defined for toggle button

### ISSUE THREAD

**Issue by zhr1130** [jakezatecky/react-checkbox-tree#275]
In the checkbox tree, state of toggle buttons are not defined . Screen reader user will be affected if state is not being announced by screen reader as user won't be get an idea of state of control.

![image](https://user-images.githubusercontent.com/18317674/118201734-fd776680-b40c-11eb-9b5e-16a22f1245c9.png)

Actual Result:
State is not defined for button control which screen reader only announces button

Expected Result:
State should define for button control so that screen reader user could get know state of control and also screen reader should announce position of items.

**Comment by jakezatecky** [comment]
Resolved with 05e73db and bec62e0.
